owncloudApp/src/main/res/values/strings.xml

```diff
@@ -219,22 +219,22 @@
     <string name="uploader_upload_succeeded_ticker">Upload succeeded</string>
     <string name="uploader_upload_succeeded_content_single">%1$s uploaded</string>
     <string name="uploader_upload_failed_ticker">Upload failed</string>
-    <string name="uploader_upload_failed_content_single">Upload of %1$s could not be completed</string>
-    <string name="uploader_upload_failed_credentials_error">Upload failed, you need to log in again</string>
-    <string name="uploader_upload_failed_ssl_certificate_not_trusted">Server certificate is not trusted</string>
-    <string name="uploader_upload_text_dialog_title">Insert a name for the new file</string>
-    <string name="uploader_upload_text_dialog_filename_error_empty">Filename must not be empty</string>
-    <string name="uploader_upload_text_dialog_filename_error_length_max">Filename must not be longer than %d characters</string>
-    <string name="uploader_upload_text_filename_hint">Filename</string>
+    <string name="uploader_upload_failed_content_single">Upload of %1$s could not be completed.</string>
+    <string name="uploader_upload_failed_credentials_error">Upload failed, you need to log in again.</string>
+    <string name="uploader_upload_failed_ssl_certificate_not_trusted">Server certificate is not trusted.</string>
+    <string name="uploader_upload_text_dialog_title">Insert a name for the new file.</string>
+    <string name="uploader_upload_text_dialog_filename_error_empty">File name must not be empty.</string>
+    <string name="uploader_upload_text_dialog_filename_error_length_max">File name must not be longer than %d characters.</string>
+    <string name="uploader_upload_text_filename_hint">File name</string>
```

Comment:
"Filename" and "file name" are both accepted, so as you prefer: https://en.wikipedia.org/wiki/Filename (we have some variables called `filename` and some others are `fileName`, so both cases in the code again)

Reply:
Again, this PR is about wording and spelling consistency. Webster doesn't accept filename as a variant yet (although it likely will soon). Variable names aren't a problem, but in the UI and docs we should be consistent and correct. https://www.merriam-webster.com/dictionary/filename